src/main/java/appeng/client/gui/implementations/GuiMEMonitorable.java

```diff
@@ -265,6 +266,15 @@ public void initGui() {
                             this.configSrc.getSetting(Settings.VIEW_MODE)));
             offset += 20;
         }
+        if (Loader.isModLoaded("ae2fc")) {
```

Comment:
Instead of checking if ae2fc is loaded, check if fluid type filters were added to the multimap

Reply:
fixed